test/fuzz/fuzzer.ml

```diff
@@ -1,49 +1,73 @@
+exception Timeout
+
 let () = Random.self_init ()
 
+let timeout_call_run (run : 'a -> unit Owi.Result.t) (m : 'a) =
```

Comment:
the `run` parameter should have type `unit -> 'a` and `timeout_call_run` should not need the `m` parameter. Then you would do:

```ocaml
  let result_unoptimized =
    let open Owi_unoptimized in
    let m = of_symbolic m in
    timeout_call_run (fun () -> run m)
  in
```